CRM/Report/Form.php

```diff
@@ -1345,14 +1345,12 @@ public function addFilters() {
 
           case CRM_Report_Form::OP_DATE:
             // build datetime fields
-            CRM_Core_Form_Date::buildDateRange($this, $fieldName, $count, '_from', '_to', ts('From:'), FALSE, $operations);
-            $count++;
+            $this->addDatePickerRange($fieldName, $field['title'], FALSE, FALSE, 'From', 'To', $operations, '_to', '_from');
             break;
 
           case CRM_Report_Form::OP_DATETIME:
             // build datetime fields
-            CRM_Core_Form_Date::buildDateRange($this, $fieldName, $count, '_from', '_to', ts('From:'), FALSE, $operations, 'searchDate', TRUE);
-            $count++;
+            $this->addDatePickerRange($fieldName, $field['title'], TRUE, FALSE, 'From', 'To', $operations, '_to', '_from');
```

Comment:
Is this breaking translation? (missing ts)

Reply:
@mlutfy i don't think so i would have thought it is picked up in https://github.com/civicrm/civicrm-core/pull/15709/files#diff-f9397c2b103c68ffb588679b5d4784a2R1356

Reply:
hmm, `ts($var)` is incorrect though. Seems like it was like that before, but it seems easy to avoid?